erpnext/hr/doctype/staffing_plan/staffing_plan.py

```diff
@@ -41,7 +41,7 @@ def set_total_estimated_budget(self):
 
 			detail.total_estimated_cost = 0
 			if detail.number_of_positions > 0:
-				if detail.vacancies > 0 and detail.estimated_cost_per_position:
+				if detail.vacancies and detail.vacancies > 0 and detail.estimated_cost_per_position:
```

Comment:
```suggestion
				if detail.vacancies and detail.estimated_cost_per_position:
```

No need to check if it's explicitly greater than 0 since the first condition will only return `True` if it's not `None` and greater than 0.